README.md

```diff
@@ -1,6 +1,6 @@
 # NEAR Indexer for Explorer
```

Comment:
Removed all (NEAR Indexer for Explorer) "Lake" suffixes as it no longer needs to be differentiated from the lake fork, it _is_ the lake fork 😄

Reply:
Removed:
* `Dockerfile`
* `.dockerignore`
* `CHANGELOG.md` (in root dir)